packages/components/src/components/progress-bar/progress-bar.tsx

```diff
@@ -71,10 +73,10 @@ export class ProgressBar {
     }
   }
 
-  transitions = (width: number) => `
+  transitions = (width: number, width_start: number) => `
     @keyframes showProgress {
       from {
-        width: 0;
+        width: ${width_start}%;
```

Comment:
Oh, variable names also need to be camelCased, sorry 🙏 — thanks why tslint is failing.

Reply:
OK, I see. While checking the tests, I also added the new property to the existing test case. 😉

Reply:
Amazing, thank you!